File: Respostas/Bee_1008/1008.c
#include <stdio.h>

int main() {
	int NUMBER, quantHorasTrab;
	float SALARY, valorHora;
	
	scanf("%d %d %f", &NUMBER, &quantHorasTrab, &valorHora);
	
	SALARY = valorHora * quantHorasTrab;
	
	printf("NUMBER = %d\n", NUMBER);
	printf("SALARY = U$ %.2f\n", SALARY);
	
	return 0;
}

File: Respostas/Bee_1008/1008.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		int NUMBER, quantHorasTrab;
		double SALARY, valorHora;
		
		Scanner input = new Scanner(System.in);
		
		NUMBER = input.nextInt();
		quantHorasTrab = input.nextInt();
		valorHora = input.nextDouble();
		
		SALARY = valorHora * quantHorasTrab;
		
		System.out.printf("NUMBER = %d\n", NUMBER);
		System.out.printf("SALARY = U$ %.2f\n", SALARY);
	}
}